internal/pkg/aws/cloudformation/cloudformation.go

```diff
@@ -27,7 +27,7 @@ var eventErrorStates = []string{
 }
 
 var waiters = []request.WaiterOption{
-	request.WithWaiterDelay(request.ConstantWaiterDelay(3 * time.Second)), // Poll for cfn updates every 3 seconds.
+	request.WithWaiterDelay(request.ConstantWaiterDelay(5 * time.Second)), // How long to wait in between poll cfn for updates.
 	request.WithWaiterMaxAttempts(1800),                                   // Wait for at most 90 mins for any cfn action.
```

Comment:
This will now wait for 2.5 hours, not 90 minutes, if the waiter delay is 5s

Reply:
The new MaxAttempts should be 1080 if we want to keep it 90 minutes max.

Reply:
Oooo thank you for catching that! fixed it